src/views/css-styles/Checked.jsx

```
@@ -0,0 +1,46 @@
+          </p>
+        </Card>
+        <Card>
+          <h2>Radio button</h2>
+          <div className="flex flex-col gap-2">
+            <div>
+              <input type="radio" name="my-input" id="yes" className="radio-yes peer"/>
+              <label for="yes" className="peer-[.radio-yes]:peer-checked:text-red-500">Yes</label>
+            </div>
+            <div>
+              <input type="radio" name="my-input" id="no" className="radio-no peer"/>
+              <label for="no" className="peer-[.radio-no]:peer-checked:text-red-500">No</label>
```

ESLint

[error] Unknown property 'for' found, use 'htmlFor' instead

(react/no-unknown-property)